### PR TITLE
Clean up json-schema ids

### PR DESCRIPTION
**Pull request by mithmatt**
### Describe your changes :

There are mixed schema uris:
https://open-metadata.org/schema
https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema

Standardize on the former

#
### Type of change :

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Backend: @sureshms @harshach @pmbrull @amiorin

### CODE REVIEW

File: catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://open-metadata.org/schema/api/data/createChart.json",
+  "$id": "https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json",
```
Comment (harshach) [inline catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json:2]: @mithmatt we did this on purpose. There is rewrite rule to redirect these schemas

Comment (mithmatt) [reply catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json]: Good to know. I'm flipping the other ones to use open-metadata.org

**Comment by mithmatt** [comment]
image


Timeout issues on pypi . need to understand further why this is happening. Saw this on other PRs earlier today
cc: @pmbrull  in case you saw this too

**Review by harshach — APPROVED** [review]

**Comment by pmbrull** [comment]
> image
> 
> Timeout issues on pypi . need to understand further why this is happening. Saw this on other PRs earlier today cc: @pmbrull in case you saw this too

This happens because we are using a big number of requirements for handling all the plugins and are a big too broad with dependencies' numbers, which forces pypi to try out a bunch of combinations until we find the right formula. Maybe we should prepare a dev vs. release set of requirements, `dev` being more flexible and `prod` everything correctly fixed with `==` to make sure all installations are the same (and faster). @harshach I'll open an issue to review this setup

**Pull request merged**